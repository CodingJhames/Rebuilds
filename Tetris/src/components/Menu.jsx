
import "./Menu.css"; 

export const Menu = ({ onClick }) => 

    <div className="Menu" >
        <button className="Button"  onClick={ onClick } >
            Play Tetris
        </button>
    </div>


