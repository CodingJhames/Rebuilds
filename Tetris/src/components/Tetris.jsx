import { useBoard } from "../hooks/useBoard";
import { Board } from "./Board";
import "./Tetris.css";


export const Tetris = ({rows, columns, setGameover }) => {
    
    const [board, setBoard] = useBoard( {  rows, columns } );


    return (
        <Board board={ board } />
    )
    
}

