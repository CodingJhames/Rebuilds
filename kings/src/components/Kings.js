
import imagenes  from '../assets/imagenes'


export const Kings = () => {

  return (
    <>
        <h1>Da Kings</h1>
        <div className='container' >
            <section className='Reyes' >
              <div className='Rey1'>
                <img src={ imagenes.img1 } alt="rey1" srcSet=""></img>
                <p>rey_ataulfo</p>
              </div>
              <div className='Rey2'>
                <img src={ imagenes.img2 } alt="rey2" srcSet=""></img>
                <p>rey_ervigio</p>
              </div>
              <div className='Rey3'>
                <img src={ imagenes.img3 } alt="rey3" srcSet=""></img>
                <p>rey_incognito</p>
              </div>
              <div className='Rey4'>
                <img src={ imagenes.img4 } alt="rey4" srcSet=""></img>
                <p>rey_recesvinto</p>
              </div>
              <div className='Rey5'>
                <img src={ imagenes.img5 } alt="rey5" srcSet=""></img>
                <p>rey_leogivildo</p>
              </div>
              <div className='Rey6'>
                <img src={ imagenes.img6 } alt="rey6" srcSet=""></img>
                <p>rey_sisebuto</p>
              </div>
              <div className='Rey7'>
                <img src={ imagenes.img7 } alt="rey7" srcSet=""></img>
                <p>rey_atanagildo</p>
              </div>
              <div className='Rey8'>
                <img src={ imagenes.img8 } alt="rey8" srcSet=""></img>
                <p>rey_teodorico</p>
              </div>
            </section>
        </div>
        
    </>
  )
}

