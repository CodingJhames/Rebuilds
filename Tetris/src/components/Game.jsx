import { Menu } from "./Menu";

export const Game = ( { rows, columns } ) => {

    const start = () => { console.log("start") };

    return (
        <div className="Game">
            <Menu  onClick={ start } />
            {/* Rows {rows} Columns {columns}  */}
        </div>

        
    )

}