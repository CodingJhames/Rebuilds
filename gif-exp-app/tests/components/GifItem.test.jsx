import { render } from "@testing-library/react";
import { GifItem } from "../../src/components/GifItem";


describe('Pruebas en GitItem', () => { 

    const title = 'Sakuragi';
    const url = 'https://slamdunk.com/sakuragi.jpg';

    test('Debe de hacer match con el snapshot', () => {

        const { container } = render( <GifItem title={ title } url={ url } /> );
        expect( container ).toMatchSnapshot();

    });





});