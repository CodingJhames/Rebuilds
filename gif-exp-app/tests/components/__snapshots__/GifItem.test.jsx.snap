// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en GitItem Debe de hacer match con el snapshot 1`] = `
<div>
  <div
    class="card"
  >
    <img
      alt="Sakuragi"
      src="https://slamdunk.com/sakuragi.jpg"
    />
    <p>
      Sakuragi
    </p>
  </div>
</div>
`;
